src/Controls/src/Core/Tweener.cs

```
@@ -39,27 +38,40 @@ public TweenerAnimation(Func<long, bool> step)
 		{
 			_step = step;
 		}
+		
 		protected override void OnTick(double millisecondsSinceLastUpdate)
 		{
 			var running = _step.Invoke((long)millisecondsSinceLastUpdate);
 			HasFinished = !running;
 		}
 
+		internal override void ForceFinish()
+		{
+			if (HasFinished)
+			{
+				return;
+			}
+
+			HasFinished = true;
+
+			// The tweeners use long.MaxValue for in-band signaling that they should
+			// jump to the end of the animation 
+			_ = _step.Invoke(long.MaxValue);
```

Should this be a FireAndForget case? Exceptions may be swallowed.